outputs.tf

```diff
@@ -130,3 +130,8 @@ output "client_ips" {
   value       = var.clients_number > 0 ? module.clients[0].client_ips : []
   description = "If 'assign_public_ip' is set to true, it will output clients public ips, otherwise private ips."
 }
+
+output "vpc_self_links" {
+  value       = length(var.subnets_name) == 0 ? module.network[0].vpc_self_links : null
```

Comment:
our module doesn't expose `vpc_self_links` and it is also not relevant to this PR

Reply:
yes, it's not relevant, but it helps a lot to plug in this module into other necessary stuff. I can extract that change into separate PR if needed.

Fixed also missing outputs in the `module/network`